caffe2/python/tutorials/py_gen/MNIST.py

```diff
@@ -10,9 +10,11 @@
 
 # # MNIST
 # 
-# In this tutorial, we will show you how to train an actual CNN model, albeit small. We will be using the good ol' MNIST dataset and the LeNet model, with a slight change that the sigmoid activations are replaced with ReLUs.
+# In this tutorial, we will show you how to train an actual convolutional neural network (a CNN) model, albeit small. We will be using the good old MNIST dataset. The problem here is hand written digits recognition. Each sample is a 28x28 picture of a hand written single digit and the label is a digit from 0 to 9. 
 # 
-# We will use the model helper - that helps us to deal with parameter initializations naturally.
+# In the LeNet model there is a a slight change that the sigmoid activations are replaced with ReLUs. There is also a flag below which allows to switch between CNN and MLP (multilayer perceptron) model architectures. 
```

Comment:
We will be constructing the LeNet model [insert link to paper] with a the sigmoid activations replaced with ReLUs [link to paper on relu benefits].  A flag below will allow us to toggle between the LeNet model and a simple MLP (multilayer perceptron) architecture.